apps/src/lib/util/AnalyticsReporter.js

```diff
@@ -106,7 +106,7 @@ class AnalyticsReporter {
     if (hostname.includes('staging')) {
       return Environments.staging;
     }
-    if (hostname.includes('localhost')) {
+    if (hostname.includes('localhost') || hostname.includes('127.0.0.1')) {
       return Environments.development;
     }
     if (hostname.includes('code.org')) {
```

Comment:
the code change seems good, but while I'm looking at this I'm wondering if the check for prod could be improved. two potential issues: 
1. `'code.org'` seems a bit too permissive, for example if someone spins up an adhoc which does not have 'adhoc' in the hostname
2. do we perhaps ever plan to use amplitude on hourofcode.com ? if so, that would be nice to include

one idea would be to match exactly on `studio.code.org` and `code.org` (and possibly `hourofcode.com`). if it turns out to be too restrictive, hopefully you'll see things show up in the unknown bucket and can use that to determine what other hostnames need to be added. https://code.org/educate/it may have some other leads for hostnames.

usual disclaimers: up to you to decide whether this is worth doing, I'm not sure how confident we are that we'll stick with amplitude and whether this kind of change is warranted yet, etc. thanks for listening :-)

Reply:
> do we perhaps ever plan to use amplitude on hourofcode.com

Not sure! Currently, I don't know of any plans but I wouldn't rule it out.

> one idea would be to match exactly on studio.code.org and code.org (and possibly hourofcode.com).

I think that makes sense to me for now. I'll leave off hourofcode for right now but match exactly on code.org and studio.code.org